packages/babel-plugin-transform-flow-strip-types/src/index.js

```diff
@@ -57,7 +57,14 @@ export default function ({ types: t }) {
           node = node.expression;
         } while (t.isTypeCastExpression(node));
         path.replaceWith(node);
-      }
+      },
+
+      ImportSpecifier(path) {
+        const { node } = path;
+        if (node.importKind !== null) {
```

Comment:
I think it'd be safer to explicitly check the types we want to exclude. Sound good?